### PR TITLE
Add test case to test whether message processor picks message from jdbc store when message body is empty

### PR DESCRIPTION
**Pull request by ThishaniLucas**
Issue : https://github.com/wso2/product-ei/issues/1449

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1204,6 +1204,12 @@
                 <artifactId>greenmail</artifactId>
                 <version>${greenmail.version}</version>
             </dependency>
+            <dependency>
+                <groupId>com.h2database</groupId>
+                <artifactId>h2</artifactId>
+                <version>${h2.db.version}</version>
```
Comment (chanikag) [inline pom.xml]: We have a h2database wso2 orbit bundle. Can't we use the same? That dependency is already there in the product-ei pom files.
            <dependency>
                <groupId>com.h2database.wso2</groupId>
                <artifactId>h2-database-engine</artifactId>
                <version>${wso2.h2.orbit.version}</version>
            </dependency>

Comment (ThishaniLucas) [reply pom.xml]: It cannot be used because the h2 version used in wos2 orbit is not supporting mv_store connection settings.
